eng/common/TestResources/README.md

```diff
@@ -28,7 +28,7 @@ is a member of multiple subscriptions.
 
 ```powershell
 Connect-AzAccount -Subscription 'YOUR SUBSCRIPTION ID'
```

Comment:
This seems odd to me that we have an internal subscription default here...  

@heaths, @pakrym:  Should we remove the default and force this to be provided?

Reply:
it's very convenient for members of our team, maybe we need to check access before defaulting. I don't know if that's possible though.